src/dhtproto/node/neo/request/core/Mixins.d

```diff
@@ -44,3 +44,38 @@ public template RequestCore ( )
         this.resources = resources;
     }
 }
+
+/*******************************************************************************
+
+    IRequestHandler-based request core mixin.
+
+*******************************************************************************/
+
+public template IRequestHandlerRequestCore ( )
+{
+    import swarm.neo.node.RequestOnConn;
+    import dhtproto.node.neo.request.core.IRequestResources;
+
+    /// Request-on-conn of this request handler.
+    private RequestOnConn connection;
+
+    /// Acquired resources of this request.
+    private IRequestResources resources;
+
+    /***************************************************************************
+
+        Passes the request-on-conn and request resource acquirer to the handler.
+
+        Params:
+            connection = request-on-conn in which the request handler is called
+            resources = request resources acquirer
+
+    ***************************************************************************/
+
+    public void initialise ( RequestOnConn connection, Object resources_object )
+    {
+        this.connection = connection;
+        this.resources = cast(IRequestResources)resources_object;
+        assert(this.resources !is null);
+    }
+}
```

Comment:
Does this needs :police_car: ?

Reply:
Hmm, traditionally, we've only included release notes for user-facing (i.e. client) changes.

Reply:
Yeah, I've just asked. I feel the same.